Sofa/framework/LinearAlgebra/src/sofa/linearalgebra/BlocFullMatrix.h

```diff
@@ -22,12 +22,4 @@
 #pragma once
 #include <sofa/linearalgebra/BlockFullMatrix.h>
 
-SOFA_DEPRECATED_HEADER("v22.12", "v23.06", "sofa/linearalgebra/BlockFullMatrix.h")
-
-namespace sofa::linearalgebra
-{
-
-template< std::size_t N, typename T>
-using BlocFullMatrix = BlockFullMatrix<N, T>;
```

Comment:
`DeprecatedAndRemoved`  is for member or variable, not types.
And in this case, whatever you put after the SOFA_DISABLED_HEADER wont be read anyway, as it will throw immediately an error at compile-time.